util/random.go

```diff
@@ -1,17 +1,56 @@
 package util
 
-import "math/rand"
+import (
+	cryptoRand "crypto/rand"
+	mathRand "math/rand"
+)
 
 // RandomString generates a random string of length n. Before calling this function, you should call
 // rand.Seed() to initialize the default source.
 //
 // Found at https://ispycode.com/Blog/golang/2016-10/How-to-generate-a-random-string-of-a-fixed-length
+//  - Adapted for more characters
 func RandomString(n int) string {
```

Comment:
I just checked the current usage of this `RandomString`, and it's used for generating signing keys. So I wonder, what's the point of having a secured version and a less secured version? I think we should keep only one version, obviously the stronger one.
We can do it in a separate PR though.

Reply:
Yes, I introduced the `RandomString` just for tests -- all usages are in unit tests. OpenId spec requires cryptographic secure randomness, so I had to introduce the other version.

I agree that we can remove this "less secure" one in another PR. Although this is supposed to be faster than the secure one, I don't think the added time to tests will be noticeable.

Reply:
Helpful to just comment code that is explicitly a test hook.